src/resources/Strings.tsx

```diff
@@ -57,13 +57,15 @@ export const Strings = {
           },
           base: {
             selectLabel: 'Base',
-            inputPlaceholder: 'Search base by ID number or author email',
+            inputPlaceholder:
+              'Search base by ID number, author email or bug number',
```

Comment:
And comment too, yes? 

`Search base by ID number, author email, bug number, or comment.`

Reply:
Same for below for new

Reply:
Alright will do

Reply:
> And author name and comment too, yes?
> 
> `Search base by ID number, author name or email, bug number, or comments.`

This text is too long for the input. Should I reduce the font size? or we could shorten the text

Reply:
What's "ID number"? if it is the revision hash, I believe we should mention that.
Also we can remove "base" and "revision", I think this makes no sense in this context.
What about:
"Search by revision ID, author, bug number and comments"

Reply:
Updated now